spectral_cube/base_class.py

```diff
@@ -108,6 +113,20 @@ def _raise_wcs_no_celestial(self):
         if not self._has_wcs_celestial:
             raise WCSCelestialError("WCS does not contain two spatial axes.")
 
+    def _celestial_axes(self):
+        '''
+        Return the spatial axes in the data from the WCS object.
+        '''
+
+        self._raise_wcs_no_celestial()
+
+        wcs_cel_axis = [self.wcs.world_axis_physical_types.index(axtype)
+                        for axtype in
+                        self.wcs.celestial.world_axis_physical_types]
+
+        # Swap to numpy ordering
+        return [wcs2np[val] for val in wcs_cel_axis][::-1]
```

Comment:
Is this line doubling down on the axis reversal with both `wcs2np[val]` and `[::-1]`?

Reply:
`wcs_cel_axis` comes out backward. I moved the `[::-1]` onto `wcs_cel_axis` so it's more clear what is reversed.

Reply:
if it comes out backward... then isn't `wcs_cel_axis` in numpy order to begin with?  Theoretically, what you're doing is more readable, but afaict, you are reversing the order twice, which is just more confusing.

It might be most helpful to include an example in the comments showing what the expected types are, e.g.:
```
# for an RA, Dec WCS we expect [1, 0] <-> [Dec, RA]
```
or something along those lines.

Reply:
There's a 2 to 0 mapping for the spatial-x axis in cubes so it's not quite just reversed. But that means this only works for 3D cubes and isn't general... so it needs to be changed regardless

Reply:
Ah!  Right, ok!